course/include/isometry.h

```diff
@@ -0,0 +1,273 @@
+#pragma once
+
+#include <cmath>
+#include <iostream>
+#include <string>
+
+namespace cppcourse {
+
+class Vector3 {
```

Comment:
Please, rebase this PR on top of you latest HEAD, so that the Vector3 code is not in this PR.